stix_shifter_modules/tanium/configuration/lang_en.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
can you make the styling format of all json files are the same? usually you can do it in vscode if you right click and format document for all the json files